src/component/parser/json/converter/ProcessConverter.ts

```diff
@@ -97,7 +95,7 @@ export default class ProcessConverter extends AbstractConverter<Process> {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   private buildFlowNodeBpmnElements(processId: string, bpmnElements: Array<any> | any, kind: ShapeBpmnElementKind): void {
     ensureIsArray(bpmnElements).forEach(bpmnElement => {
-      if (kind as BpmnEventKind) {
+      if (this.isEvent(kind)) {
```

Comment:
:question: what is the advantage to remove the type?

Reply:
type is preserved in function param
I removed kind as BpmnEventKind as it resulted in always true and made all our elements of type Event

Reply:
:heavy_check_mark: this is now clear